source/frontend/src/features/projects/reports/ProjectExportForm.tsx

```diff
@@ -0,0 +1,62 @@
+import { Form, Formik } from 'formik';
+import * as React from 'react';
+
+import { Select, SelectOption } from '@/components/common/form';
+import { SectionField } from '@/components/common/Section/SectionField';
+import { Api_Person } from '@/models/api/Person';
+import { Api_Project } from '@/models/api/Project';
+import { Api_ExportProjectFilter } from '@/models/api/ProjectFilter';
+
+import { ExportProjectModel, ProjectExportTypes } from './models';
+import { ProjectExportFormContent } from './ProjectExportFormContent';
+
+export interface IProjectExportFormProps {
+  onExportTypeSelected: () => void;
+  onExport: (filter: Api_ExportProjectFilter) => void;
```

Comment:
nit: we might update this later but likely the onExport should be an async function